src/main/java/com/google/sps/data/Attendee.java

```diff
@@ -0,0 +1,92 @@
+package com.google.sps.data;
+
+import java.util.Date;
+import com.google.appengine.api.datastore.Entity;
+import com.google.sps.data.EntityConstants;
+
+/** Class that represents a user in the session as an attendee. */
+public class Attendee implements AttendeeInterface {
+
+  // The id of the session this attendee is in.
+  private final String sessionId;
+
+  // The screen name of this attendee.
+  private final String screenName;
+
+  // Time the attendee last polled the server.
+  private Date timeLastPolled;
+  
+  /** Initializes an Attendee object
+   * @param {String} sessionId - the session the attendee is attached to.
+   * @param {String} screenName - the screen name of the attendee. 
+   * @param {Date} timeLastPolled - the time attendee was last polled.
+   */
+  public Attendee (String sessionId, String screenName, Date timeLastPolled) {
+    this.sessionId = sessionId;
+    this.screenName = screenName;
+    this.timeLastPolled = timeLastPolled;
+  }
+
+  public String getSessionId() {
+    return sessionId;
+  }
+
+  public String getScreenName() {
+    return screenName;
+  }
+
+  public Date getTimeLastPolled() {
+    return timeLastPolled;
+  }
+  
+  @Override
+  public boolean equals(Object obj) {
+    // self check
+    if (this == obj)
+      return true;
+    // null check
+    if (obj == null)
+      return false;
+    // type check
+    if(obj.getClass() != getClass())
+      return false;
```

Comment:
I know this is a convenience thing but please dont do this. This will inevitably lead to weird bugs. Just use the full if-block syntax:
if (blah) {
return true;
}
instead of the shorthand:
if (blah)
return true;